src/mixins/itext_key_behavior.mixin.js

```diff
@@ -116,11 +120,24 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       return;
     }
     var keyMap = this.direction === 'rtl' ? this.keysMapRtl : this.keysMap;
-    if (e.keyCode in keyMap) {
-      this[keyMap[e.keyCode]](e);
+    var func;
+    if (e.key in keyMap) {
+      func = keyMap[e.key];
+    }
+    else if (e.keyCode in keyMap) {
+      func = keyMap[e.keyCode];
+    }
+    else if ((e.key in this.ctrlKeysMapDown) && (e.ctrlKey || e.metaKey)) {
+      func = this.ctrlKeysMapDown[e.key];
     }
     else if ((e.keyCode in this.ctrlKeysMapDown) && (e.ctrlKey || e.metaKey)) {
-      this[this.ctrlKeysMapDown[e.keyCode]](e);
+      func = this.ctrlKeysMapDown[e.keyCode];
+    }
+    if (typeof func === 'string' && typeof this[func] === 'function') {
+      func = this[func];
+    }
+    if (typeof func === 'function') {
+      func.call(this, e);
```

Comment:
no let's do that if someone needs a function that does not exist, it adds it to the class and continue to call it by name.

Reply:
Why should it add it to the class?
That's strange.

Reply:
well right now the map is:
```
{
  keycode: functionName
}
```
after can be:

```
{
  keycode: functionName
  otherkeycode: (e) => { ...doSomething }
}
```

So you have:

IText.prototype.ctrlKeysMapUp.otherKeyCode that is a function.  You added it to the class. But as name it has the keycode.

today you have:

IText.prototype.ctrlKeysMapUp.'65' = 'selectAll' and then you have IText.prototype.selectAll

the function is added to the prototype and its name to the map.
Those approach are totally similar, one is not more dynamic of the other, and so i don't see the reason to have both options.